app/models/manager_refresh/inventory_collection_default/infra_manager.rb

```diff
@@ -42,8 +57,8 @@ def guest_devices(extra_attributes = {})
     def hardwares(extra_attributes = {})
       attributes = {
         :model_class => ::Hardware,
-        :manager_ref => [:vm_or_template],
-        :association => :hardwares
+        :manager_ref => [:vm_or_template], # :host
```

Comment:
Thanks, will check this approach and if it works I would go the same with `operating_system`